.changelog/2463.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+resource/mongodbatlas_alert_configuration: Fixes an issue where the `terraform apply` command crashes if you attempt to edit an existing `mongodbatlas_alert_configuration` by adding a value to `threshold_config`.
```

Comment:
```suggestion
resource/mongodbatlas_alert_configuration: Fixes an issue where the `terraform apply` command crashes if you attempt to edit an existing `mongodbatlas_alert_configuration` by adding a value to `threshold_config`
```

don't finish in period, e.g.: https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CHANGELOG.md

same for https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2462 in case you can also change its changelog period at the end

Reply:
Okay, removing this period and other unreleased entries periods for consistency. Changelog entries will be updated once this is merged. FYI @JuliaMongo